### PR TITLE
Add RawCMS C# Project

### PR DESCRIPTION
**Pull request by zeppaman**
Please complete the following checklist if your PR is adding new link to the list:

- [X] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [X] This PR does not introduce duplicates to the list
- [X] The link is added at the bottom of the list category
- [X] Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
- [X] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

**Comment by zeppaman** [comment]
there is something I can do on my side to avoid that this issue will be closed? the issues is yet marked as stale and going to be closed, according to the previous comment.

**Comment by zeppaman** [comment]
there is something I can do on my side to avoid that this issue will be closed? the issues is yet marked as stale and going to be closed, according to the previous comment.

**Comment by MunGell** [comment]
Hi @zeppaman 

Could you please fix the link (see guidelines) and consider changing label to good first issue, which is also present in the repository?

**Comment by zeppaman** [comment]
Hi @MunGell I changed the label into the link. Link is now:
```
- [RawCMS](https://github.com/arduosoft/RawCMS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> RawCMS is a headless CMS written in ASP.NET Core, built for developers that embrace API-first technology.
```

and should be in the format:
```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 
 - [Shouldly](https://github.com/shouldly/shouldly/labels/Jump-In) _(label: Jump-In)_ <br> Should testing for .net - the way Asserting *Should* be!
 - [MvvmCross](https://github.com/MvvmCross/MvvmCross/labels/first-timers-only) _(label: first-timers-only)_ <br> The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
+- [RawCMS](https://github.com/arduosoft/RawCMS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> RawCMS is a headless CMS written in ASP.NET Core, built for developers that embrace API-first technology.
```
Comment (MunGell) [inline README.md]: I believe the link here should be https://github.com/arduosoft/RawCMS/labels/good%20first%20issue -  a direct link to the label

File: README.md [hunk 2]
```diff
@@ -49,6 +50,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [OpenRA](https://github.com/OpenRA/OpenRA/labels/Easy) _(label: Easy)_ <br> A Real Time Strategy game engine supporting early Westwood classics such as Command & Conquer: Red Alert.
 - [osu!](https://github.com/ppy/osu/labels/good%20first%20issue) _(label: good first issue)_ <br> Music game. Rhythm is just a *click* away!
 
+
```
Comment (MunGell) [inline README.md]: Please remove this extra line

File: README.md (hunk 1 above)
Comment (zeppaman) [reply README.md]: Changed.

File: README.md (hunk 2 above)
Comment (zeppaman) [reply README.md]: Removed

**Pull request merged**